src/pages/search.tsx

```diff
@@ -158,17 +158,36 @@ const Search: NextPage<SearchProps> = ({ translations, chaptersData }): JSX.Elem
     [],
   );
 
+  // a ref to know weather this is the initial search request made when the user loads the page or not
+  const isInitialSearch = useRef(true);
+
   // listen to any changes in the API params and call BE on change.
   useEffect(() => {
     // only when the search query has a value we call the API.
     if (debouncedSearchQuery) {
-      getResults(debouncedSearchQuery, currentPage, selectedTranslations, selectedLanguages);
+      // we don't want to reset pagination when the user reloads the page with a ?page={number} in the url query
+      if (!isInitialSearch.current) setCurrentPage(1);
```

Comment:
not-picking, we have been using curly brackets even for a single-line condition.
```
if (!isInitialSearch.current) {
  setCurrentPage(1);
}
```

Reply:
Maybe we should add an eslint rule ?